.github/workflows/update-cli-reference.yml

```diff
@@ -0,0 +1,46 @@
+on:
+  workflow_dispatch
+    
+jobs:
+  update-cli-reference:
+    name: Update the CLI reference
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: Download and prepare the am binary
+        run: |
+          mkdir /tmp
+          cd /tmp/
+          curl -L https://github.com/autometrics-dev/am/releases/latest/download/am-linux-x86_64 -o am
+          chmod u+x am
+
+      - name: Generate the CLI reference
+        run: |
+          /tmp/am markdown-help > src/pages/local-development/cli-reference.md
+          echo "am_version=$(/tmp/am --version | awk '{print $2}')" >> "$GITHUB_ENV"
+
+      - name: Cleanup
+        run: rm -r /tmp
+
+      - name: Create a branch and add the changed commit
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
What is that ID in there?

Reply:
It's the id of the github actions bot user (so that a nice icon shows up). However, removed it as redundant per this answer: https://github.com/actions/checkout/discussions/479